src/main/java/org/jenkinsci/plugins/saml/SamlSecurityRealm.java

```diff
@@ -73,6 +75,36 @@
   private static final String DEFAULT_USERNAME_CASE_CONVERSION = "none";
   private static final String DEFAULT_EMAIL_ATTRIBUTE_NAME = "email";
 
+  static {
+    runUberClassLoaderAsContextClassLoader(new com.google.common.base.Supplier<Void>() {
+      @Override
+      public Void get() {
+        try {
+          InitializationService.initialize();
+        } catch (InitializationException e) {
+          LOG.log(Level.SEVERE, "Could not initialize opensaml service.", e);
+        }
+        return null;
+      }
+    });
+  }
+
+  // opensaml uses java.util.ServiceLoader to find service implementations. Implementations from other jars cannot
+  // be found using plugin's default context classloader.
+  private static <F, T> T runUberClassLoaderAsContextClassLoader(@Nonnull com.google.common.base.Supplier<T> runnable) {
```

Comment:
I am not sure to understand the goal of this code, I think that it is not needed to load the openSAML library bundle